include.js

```diff
@@ -1682,6 +1682,14 @@ let cards = [
     author: 'Aditya',
     githubLink: 'https://github.com/adityaanandz'
   },
+
+  {
+    artName: 'flower-animation',
+    pageLink: './Art/flower-animation/index.html',
```

Comment:
This needs to correspond with the folder name, e.g. `Art/saurabhsingh720/index.html`